test/e2e/flowaggregator_test.go

```diff
@@ -264,20 +279,53 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 						}
 					}
 				}
-				if checkNetworkPolicy {
+				if checkK8sNetworkPolicy {
 					// Check if records have both ingress and egress network policies.
 					if !strings.Contains(record, ingressNetworkPolicyName) {
 						t.Errorf("Record does not have NetworkPolicy name with ingress rule")
 					}
 					if !strings.Contains(record, fmt.Sprintf("%s: %s", "ingressNetworkPolicyNamespace", testNamespace)) {
```

Comment:
I don't know why we pass the IE name as a parameter to the string formatting function, instead of:
```go
fmt.Sprintf("ingressNetworkPolicyNamespace: %s", testNamespace)
```

same below

Reply:
updated